CORE-8421 Remove kameleon db version checks

psarando opened the pull request:
Since entities have been moved out of kameleon, facepalm no longer needs to validate the version of kameleon against the current db version.

This PR also cleans up kameleon dependencies as part of the cyverse-de/kameleon#6 effort.

psarando committed:
CORE-8421 Bump kameleon dependency version to 2.8.4-SNAPSHOT

psarando committed:
CORE-8421 Clean up kameleon dependencies in facepalm.core.

ianmcorvidae reviewed (approved):
This looks straightforward, I say :shipit:

psarando commented:
Thanks for the review 👍 
I'll keep this open until cyverse-de/kameleon#6 is merged and kameleon has a new release version.
Then I'll bump that version in facepalm and merge the PR.

psarando committed:
CORE-8421 Remove references to kameleon.core

Since entities have been moved out of kameleon, facepalm no longer needs
to validate the version of kameleon against the current db version.
Also removed unused facepalm.error-codes/error-exit function.

psarando commented:
Now that entities have been moved out of kameleon, I was able to include a change that allows facepalm to no longer require the db version in kameleon (it no longer needs to be validated against the current db version).

So it would be great to also get a review of this change from someone.

ianmcorvidae commented:
I looked at the added commit and it looks good to me!

slr71 reviewed (approved):
Looks good to me. :shipit:

psarando committed:
CORE-8421 Bump version to 2.10.0-SNAPSHOT.

psarando committed:
CORE-8421 Bump kameleon dependency to release version 3.0.0.

psarando commented:
Thanks for the reviews! 👍
